runtime/src/main/java/org/corfudb/protocols/wireprotocol/StateReq.java

```diff
@@ -0,0 +1,22 @@
+package org.corfudb.protocols.wireprotocol;
+
+import io.netty.buffer.ByteBuf;
+
+public class StateReq implements ICorfuPayload<StateReq> {
```

Comment:
Do you need this class? You can replace this with just using CorfuMsg.

Reply:
No, removed.